vendor/bat-native-ads/include/bat/ads/new_tab_page_ad_wallpaper_info.h

```diff
@@ -8,13 +8,12 @@
 
 #include <vector>
 
-#include "bat/ads/export.h"
 #include "bat/ads/new_tab_page_ad_wallpaper_focal_point_info.h"
 #include "url/gurl.h"
 
 namespace ads {
 
-struct ADS_EXPORT NewTabPageAdWallpaperInfo final {
```

Comment:
why do we remove export here? will it break the component build?

Reply:
Adding this back and will push the change soon.

Reply:
Fixed